src/pybind/mgr/cephadm/http_server.py

```diff
@@ -20,7 +21,7 @@ def cherrypy_filter(record: logging.LogRecord) -> int:
 
 
 logging.getLogger('cherrypy.error').addFilter(cherrypy_filter)
-cherrypy.log.access_log.propagate = False
+cherrypy.log.access_log.propagate = True
```

Comment:
I think I enabled it during the testing. We had some discussions to make it configurable so we can enable it in case we need to. By now I'll keep it disabeld.